stable_baselines/deepq/experiments/run_atari.py

```diff
@@ -18,8 +18,6 @@ def main():
     parser.add_argument('--dueling', type=int, default=1)
     parser.add_argument('--prioritized-replay-alpha', type=float, default=0.6)
     parser.add_argument('--num-timesteps', type=int, default=int(10e6))
-    parser.add_argument('--checkpoint-freq', type=int, default=10000)
```

Comment:
nice catch, at some we should get rid of those files anyway now that we have the zoo